testdata/e2e/runTestValidate.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+curl --version
+
+echo "Test /api/validate should respond 200 if sealed secret is valid"
+
+SEALED_SECRET=$(curl --silent --show-error --request POST 'http://localhost/ssw/api/kubeseal' \
+  --header 'Accept: application/x-yaml' \
+  --data-binary '@stringData.yaml')
+
+echo "$SEALED_SECRET" | yq -r .apiVersion | grep --quiet "bitnami.com/v1alpha1"
+echo "$SEALED_SECRET" | yq -r .kind | grep --quiet "SealedSecret"
+echo "$SEALED_SECRET" | yq -r .metadata.name | grep --quiet "mysecretname"
+echo "$SEALED_SECRET" | yq -r .metadata.namespace | grep --quiet "mysecretnamespace"
+
+RESPONSE=$(curl --silent --show-error --request POST 'http://localhost/ssw/api/validate' \
```

Comment:
the test fails probably here: https://github.com/bakito/sealed-secrets-web/actions/runs/4962251600/